src/core/Akka.API.Tests/CoreAPISpec.ApproveCluster.approved.txt

```diff
@@ -437,11 +437,12 @@ namespace Akka.Cluster.SBR
 }
 namespace Akka.Cluster.Serialization
 {
-    public class ClusterMessageSerializer : Akka.Serialization.Serializer
+    [Akka.Annotations.InternalApiAttribute()]
+    public class ClusterMessageSerializer : Akka.Serialization.SerializerWithStringManifest
```

Comment:
So even though this is part of the public API, I don't think this is much of a "breaking change" as the serializer isn't called directly from user code (that I'm aware of) and it's really meant to be an internal api.